docs/CONFIG-VARS.md

```diff
@@ -209,6 +209,7 @@ Custom policy:
 | autoscaling_enabled | Enable cluster autoscaling | bool | true | |
 | ssh_public_key | File name of public ssh key for jump and nfs VM | string | "~/.ssh/id_rsa.pub" | Required with `create_jump_vm=true` or `storage_type=standard` |
 | cluster_api_mode | Public or private IP for the cluster api| string|"public"|Valid Values: "public", "private" |
+| enable_multi_zone | Set to true to deploy EKS Node Pools in multiple availability zones | boolean | false | **WARNING: changing this from true to false after infrastructure creation is destructive** |
```

Comment:
In addition to the note that moving from true to false is destructive, I believe it would be helpful to add the recommendation for using backup and restore that you made in the Changes section of this PR for folks that want to change from true to false.

Reply:
Good idea, added a link to the backup and restore doc for that variable in the notes section.

I'll also make a note to include something about toggling that `enable_multi_zone`  flag and how it can be destructive.